core/dbt/flags.py

```diff
@@ -170,7 +172,13 @@ def set_from_args(args, user_config):
     USE_EXPERIMENTAL_PARSER = get_flag_value("USE_EXPERIMENTAL_PARSER", args, user_config)
     VERSION_CHECK = get_flag_value("VERSION_CHECK", args, user_config)
     WARN_ERROR = get_flag_value("WARN_ERROR", args, user_config)
-    WARN_ERROR_OPTIONS = get_flag_value("WARN_ERROR_OPTIONS", args, user_config)
+
+    warn_error_options_str = get_flag_value("WARN_ERROR_OPTIONS", args, user_config)
```

Comment:
Which enum type are you referring to? `get_flag_value` accepts a string as the first argument in this line similar to how it works for all the other `get_flag_value` calls above.

Reply:
resolved: it would be nice for auto-completion purposes to have the first argument have been an enum here, but given that [we're moving away from this file](https://github.com/dbt-labs/dbt-core/issues/6701), a refactor at this point isn't worth the effort.